File: docs/snippets/all/archetypes/gps_coordinates.cpp
// Log some very simple points.

#include <rerun.hpp>

int main() {
    const auto rec = rerun::RecordingStream("rerun_example_gps_coordinates");
    rec.spawn().exit_on_failure();

    rec.log("points", rerun::GpsCoordinates({{47.6343f, 19.1397f, 0.0f}, {47.6344f, 19.1395f, 1.0f}}));
}

File: docs/snippets/all/archetypes/gps_coordinates.rs
//! Log some very simple points.

fn main() -> Result<(), Box<dyn std::error::Error>> {
    let rec = rerun::RecordingStreamBuilder::new("rerun_example_points3d").spawn()?;

    rec.log(
        "points",
        &rerun::GpsCoordinates::new([(47.6344, 19.1397, 0.0), (47.6344, 19.1399, 1.0)]),
    )?;

    Ok(())
}
